src/include/processor/operator/ddl/create_table.h

```diff
@@ -8,12 +8,17 @@ namespace processor {
 class CreateTable : public DDL {
 public:
     CreateTable(PhysicalOperatorType operatorType, Catalog* catalog, string tableName,
-        vector<PropertyNameDataType> propertyNameDataTypes, uint32_t id, const string& paramsString)
-        : DDL{operatorType, catalog, id, paramsString}, tableName{std::move(tableName)},
-          propertyNameDataTypes{move(propertyNameDataTypes)} {}
+        vector<PropertyNameDataType> propertyNameDataTypes, const DataPos& outputPos, uint32_t id,
+        const string& paramsString)
+        : DDL{operatorType, catalog, outputPos, id, paramsString}, tableName{std::move(tableName)},
+          propertyNameDataTypes{std::move(propertyNameDataTypes)} {}
 
     virtual ~CreateTable() override = default;
 
+    void executeDDLInternal() override = 0;
```

Comment:
I think we don't need to explicitly declare this is a pure virtual function in the child class.